roles/monitoring_sap/tasks/main.yml

```diff
@@ -3,20 +3,21 @@
 # these Ansible tasks are NOT idempotent on purpose due to the actions:
 # existing configurations are overwritten on purpose
 
-#################################################################
-#  Block: Delete a SAP System from the monitoring configuration
-#################################################################
+########################################################################################
+#  Block: Delete one SAP monitoring configuration with the specific  <sap_monitoring_nr>
+########################################################################################
+
+- name: Delete the SAP monitoring configuration with the specific <sap_monitoring_nr>
 
-- name: Delete a SAP System from the monitoring configuration
   when: sap_monitoring_action == "delete"
   block:
 
-    - name: Check existence of the variables defined in ./vars/sample-monitoring-sap-parameters.yml
+    - name: Check existence of the SAP and monitoring variables
```

Comment:
Rename this task: Assert required variables are defined for removing SAP monitoring configuration

Reply:
THis task needs to be renamed just like the one after